students/eunhyekim/users/views.py

```diff
@@ -1,10 +1,30 @@
-import json, re
+import json
 
-from django.http      import JsonResponse
-from django.views     import View
-from users.models     import User
+from django.http                import JsonResponse
+from django.views               import View
+from django.core.exceptions     import ValidationError
 
+from users.models               import User
+from users.validation           import Validation_Email, Validation_Password
+
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data     = json.loads(request.body)
+            password = User.objects.get(email = data["email"]).password
+            
+            if data['password'] != password:
+                return JsonResponse({'message':'INVALID_USER'}, status = 400)
```

Comment:
로그인 시 불필요하게 변수를 지정할 필요가 없어 보입니다. 그저 request를 통해 받아온 email과 password를 동시에 만족하는 경우만 핸들링 해주면 될 것 같습니다! 
```suggestion
            if not User.objects.filter(email = data["email"], password = data["password"]).exists():
	        return JsonResponse({'message' : 'INVALID_USER'}, status=401)
```

Reply:
네 이해하였습니다! 수정했습니다!